GUIDE.md

```diff
@@ -0,0 +1,81 @@
+Pegaswitch Getting Started Guide
+================================
+A guide to getting your Switch from factory to running Pegaswitch.
+
+Requirements
+------------
+- A Nintendo Switch running FW 1.0.0 - 3.1.0
+	- For 1.0.0, you will also need a copy of [Puyo Puyo Tetris JP](https://www.amazon.com/gp/product/B01N4PKS4S/ref=oh_aui_detailpage_o00_s00?ie=UTF8&psc=1)
+- A PC running any OS capable of running Node.JS that is connected to a network your Switch can connect to
+
+Setting up the Environment
+--------------------------
+### Windows
+Windows users will need to setup the Windows Subsystem for Linux (WSL). Instructions on how to set that up can be found [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10). After WSL has been setup, continue with the Linux instructions, using the Ubuntu/Debian commands.
```

Comment:
`setup` -> `set up`, and perhaps remove the "Instructions on how to set that up..." sentence and make "setup the Windows Subsystem for Linux (WSL)" be the link instead